packages/babel-types/src/definitions/experimental.js

```diff
@@ -61,23 +61,3 @@ defineType("ExportNamespaceSpecifier", {
     }
   }
 });
-
-defineType("RestProperty", {
```

Comment:
You want to do this virtual types in 6.x? imho it doesn't make sense to introduce bc code in a major version.